bool x_gripperOpen
bool x_gripperClose
